From cdf8c9e87a43bc3700f6f2db79fa2b52e5b9f1d2 Mon Sep 17 00:00:00 2001
From: dzm <dengzhimao@alientek.com>
Date: Sat, 24 Feb 2024 17:21:02 +0800
Subject: [PATCH] Fixed an issue where the app screen would go black when
 switching focus

---
 libweston/desktop/surface.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libweston/desktop/surface.c b/libweston/desktop/surface.c
index 6f1848f..8c7c820 100644
--- a/libweston/desktop/surface.c
+++ b/libweston/desktop/surface.c
@@ -897,7 +897,7 @@ weston_desktop_surface_set_attrs(struct weston_desktop_surface *surface,
 			     strlen(SURFACE_ATTR_ALPHA))) {
 			double alpha = atof(p + strlen(SURFACE_ATTR_ALPHA));
 
-			weston_surface_set_alpha(wsurface, alpha);
+			/*weston_surface_set_alpha(wsurface, alpha);*/
 		} else {
 			weston_log("%s: warning: unsupported attr: %s\n",
 				   __func__, p);
-- 
2.25.1

